File: sdkwork-ai-sdk-java/src/main/java/com/sdkwork/ai/api/ModerationsApi.java
package com.sdkwork.ai.api;

import com.sdkwork.ai.http.HttpClient;
import com.sdkwork.ai.model.*;
import java.util.List;
import java.util.Map;

public class ModerationsApi {
    private final HttpClient client;
    
    public ModerationsApi(HttpClient client) {
        this.client = client;
    }

    /** Create moderation */
    public ModerationResponse createModeration(ModerationRequest body) throws Exception {
        return (ModerationResponse) client.post(ApiPaths.aiPath("/moderations"), body);
    }
}

File: sdkwork-ai-sdk-kotlin/src/main/kotlin/com/sdkwork/ai/api/ModerationsApi.kt
package com.sdkwork.ai.api

import com.sdkwork.ai.*
import com.sdkwork.ai.http.HttpClient

class ModerationsApi(private val client: HttpClient) {

    /** Create moderation */
    suspend fun createModeration(body: ModerationRequest): ModerationResponse? {
        return client.post(ApiPaths.aiPath("/moderations"), body) as? ModerationResponse
    }
}
